designer/apps/pages/serializers.py

```diff
@@ -0,0 +1,64 @@
+""" Serialzers for Page Models """
+from rest_framework import serializers
+from wagtail.wagtailcore.models import Page
+from designer.apps.pages.models import IndexPage, ProgramPage
+from designer.apps.branding.serializers import BrandingField
+
+
+class PageSerializer(serializers.ModelSerializer):
```

Comment:
As per above, I think all the serializers should go in the `api` app.

Reply:
will do